### PR TITLE
Added `image_url/1` to Emoji module

### PR DESCRIPTION
**Pull request by skippi**
* We should add this because it makes getting an emoji's image URL less cumbersome.

### CODE REVIEW

File: lib/nostrum/struct/emoji.ex [hunk 1]
```diff
@@ -156,6 +156,36 @@ defmodule Nostrum.Struct.Emoji do
...
+  iex> Nostrum.Struct.Emoji.image_url(emoji)
+  nil
+  ```
+  """
+  @spec image_url(t) :: String.t() | nil
+  def image_url(emoji)
+  def image_url(%__MODULE__{id: nil}), do: nil
+
+  def image_url(%__MODULE__{animated: true, id: id}),
+    do: "https://cdn.discordapp.com/emojis/#{id}.gif"
```
Comment (Kraigie) [inline lib/nostrum/struct/emoji.ex]: I think we should move these URLs to the constants folder.

**Comment by skippi** [comment]
Putting on hold until #82 is complete.

File: lib/nostrum/struct/emoji.ex (hunk 1 above)
Comment (skippi) [reply lib/nostrum/struct/emoji.ex]: Addressed in 0dac258.

**Comment by skippi** [comment]
Ready for more code review.

**Comment by Kraigie** [comment]
Thanks!

**Pull request merged**